content/source/docs/enterprise/upgrade/index.html.md

```diff
@@ -10,85 +10,74 @@ If you used the legacy version of Terraform Enterprise (TFE), you probably have
 
 Follow these steps to migrate your old TFE environments to new TFE workspaces.
 
-## 1. Create a New Organization
+~> **Important:** Workspace migration imports the settings and the most recent Terraform state from a legacy environment. It does not import the state history or the run event history. If you need to preserve historical data from your legacy environments for regulatory reasons, please contact Hashicorp support for assistance and do not delete your legacy environments after migrating.
```

Comment:
Hashicorp -> HashiCorp (and throughout)

Reply:
OOPS D:

Reply:
Rebased and fixed, wow that's an embarrassing one. I blame Bitbucket for giving me bad reflexes. 😅